scripts/wasm-rebuild/docker-scripts/rebuild_current.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+# This script is expected to be run inside the docker image trzeci/emscripten:sdk-tag-1.39.3-64bit and
+# be called by ./rebuild_tags.sh.
+
+echo "========== STAGE 1: PREPARE ========== ($(date))"
+git rev-parse --short=8 HEAD > commit_hash.txt
+echo -e "" > prerelease.txt
+sed -i -e 's/-Wl,--gc-sections//' cmake/EthCompilerSettings.cmake
+sed -i -e 's/-Werror/-Wno-error/' cmake/EthCompilerSettings.cmake
```

Comment:
This is by the way only needed for ``< 0.5.0`` - not sure if we should investigate this further - my guess is that the newer compiler will complain about more stuff that wasn't addressed at the time, so I'd say it makes sense to tell the compiler to be lenient - I'd say it's preferable to attempting to patch it, since so far we can cope without any source changes.